### PR TITLE
 🐦[FSharp] Falco - v2.0.3

### PR DESCRIPTION
**Pull request by pimbrouwers**
Upgrade to Falco v2.0.3

### CODE REVIEW

**Comment by waghanza** [comment]
@pimbrouwers could you merge or rebase from master ? I've changed a lot of things

**Comment by pimbrouwers** [comment]
More than happy to do that Marwan!

**Comment by pimbrouwers** [comment]
@waghanza This should be good to go.

File: fsharp/falco/config.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 framework:
   github: pimbrouwers/Falco
-  version: 1.2
+  version: 2.0.3
```
Comment (waghanza) [inline fsharp/falco/config.yaml:3]: could you use `X.Y` form ?

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by pimbrouwers** [comment]
Done!

**Review by waghanza — APPROVED** [review]

**Comment by pimbrouwers** [comment]
See #3455

**Pull request merged**